Change Repository Access Reccomendations in Manual-Install Guide (#7859)
fishtown-analytics/docs.getdbt.com: website/docs/guides/manual-install-qs.md

The following steps use [GitHub](https://github.com/) as the <Constant name="git" /> provider for this guide, but you can use any <Constant name="git" /> provider. You should have already [created a GitHub account](https://github.com/join).

1. [Create a new GitHub repository](https://github.com/new) named `dbt-tutorial`.
2. Select:
    - **Private (recommended):** To secure your environment and prevent private information (like credentials) from being public.
    - **Public:** If you need to easily collaborate and share with others, especially outside of your organization. 

> You can always change the privacy settings later. 

Reviewer: ```suggestion

You can always change the privacy settings later. 
```

The following steps use [GitHub](https://github.com/) as the <Constant name="git" /> provider for this guide, but you can use any <Constant name="git" /> provider. You should have already [created a GitHub account](https://github.com/join).

1. [Create a new GitHub repository](https://github.com/new) named `dbt-tutorial`.

> 2. Select:

Reviewer: ```suggestion
2. Select one of the following (You can always change this setting later):
```

    - **Private (recommended):** To secure your environment and prevent private information (like credentials) from being public.
    - **Public:** If you need to easily collaborate and share with others, especially outside of your organization. 

> You can always change the privacy settings later. 

Reviewer: ```suggestion
```

The following steps use [GitHub](https://github.com/) as the <Constant name="git" /> provider for this guide, but you can use any <Constant name="git" /> provider. You should have already [created a GitHub account](https://github.com/join).

1. [Create a new GitHub repository](https://github.com/new) named `dbt-tutorial`.

> 2. Select **Private**. This is so you don't accidentially share any credentials for your Google Cloud account. You can always make it public later.

Reviewer: Good call out but let's split the difference and highlight both options to give the big picture:
```suggestion
2. Select:
    - **Private (recommended):** To secure your environment and prevent private information (like credentials) from being public.
    - **Public:** If you need to easily collaborate and share with others, especially outside of your organization. 
You can always change the privacy settings later. 
```
